lib/locales/hu.ts

```diff
@@ -0,0 +1,936 @@
+import { Locale } from '../types/locale';
+
+export const locale: Locale = {
+  id: 'hu',
+
+  seasons: {
+    advent: {
+      season: 'Adventi idő',  // src: https://tar.liturgia.hu/Naptar/DIR2022_JAV_2021-12-13.pdf#page=29
+              weekday: 'Advent {{week, romanize}}. hete $t(weekdays:{{dow}})',  // src: https://kattima.hu/hu/app/pray/bences-zsolozsma-l-a-u-d-e-s-advent-ii.-hete-hetfo-80378
+      sunday: 'Advent {{week, romanize}}. vasárnapja',  // src: https://tar.liturgia.hu/Naptar/DIR2022_JAV_2021-12-13.pdf#page=30
+      privileged_weekday: '$t(months:11, capitalize) {{day}}.',  // src: https://kattima.hu/hu/app/pray/bences-zsolozsma-l-a-u-d-e-s-december-17.-szombat-80429
+    },
+
+    christmas_time: {
+      season: 'Karácsonyi idő',  // src: https://tar.liturgia.hu/Naptar/DIR2022_JAV_2021-12-13.pdf#page=40
+      day: '$t(weekdays:{{dow}}, capitalize) karácsonyi időben',
+      octave: 'Karácsony nyolcada alatti $t(ordinals:{{count}}, romanize). nap',  // src: https://tar.liturgia.hu/Naptar/DIR2022_JAV_2021-12-13.pdf#page=43
+      second_sunday_after_christmas: 'Karácsony utáni II. vasárnap',  // src: https://tar.liturgia.hu/Naptar/DIR2022_JAV_2021-12-13.pdf#page=46
+      before_epiphany: '$t(months:0, capitalize) {{day}}.',
+      after_epiphany: '$t(weekdays:{{dow}}, capitalize) az Úrunk megjelenése után',
+    },
+
+    ordinary_time: {
+      season: 'Évközi idő',  // src: https://tar.liturgia.hu/Naptar/DIR2022_JAV_2021-12-13.pdf#page=50
+      weekday: '$t(weekdays:{{dow}}, capitalize) évközi időben, {{week, romanize}}. hét',  // src: https://kattima.hu/hu/app/pray/bences-zsolozsma-k-o-m-p-l-e-t-o-r-i-u-m-csutortok-evkozi-idoben-i.-het-77548
```

Comment:
@MMate2007, this one sounds weird to me, however, I could not (yet) find a better source for this string.

Reply:
Please give me an example of what it will display.

Reply:
`Hétfő évközi időben, I. hét`.

See [this comment of mine](https://github.com/romcal/romcal/pull/403#discussion_r1316094197).

Reply:
It seems right to me.

Reply:
Grammatically, yes, but is it actually used this way? In `hu_mr_2022_ed3` (which I have OCR’d and [optimised](https://drive.google.com/drive/u/3/folders/1F4UwlJ-ajcd_VfeBBiGgMJwKD2Kyj_7T), but forgot to tell you about it) does not use this celebration name in full (it is simply `Hétfő`/`Monday`), therefore this might need to be verified using other liturgical books (e.g. lectionary).